scripts/build/builders/host.py

```diff
@@ -352,6 +358,12 @@ def __init__(self, root, runner, app: HostApp, board=HostBoard.NATIVE,
             self.build_command = 'chip-repl'
 
     def GnBuildArgs(self):
+        if self.app == HostApp.SIMULATED_APP1:
+            self.extra_gn_options.extend(
```

Comment:
```python
self.extra_gn_options.append('chip_tests_zap_config="app1"')
```

seems shorter.

Reply:
why are we doing this here and not in the constructor (like we do for crypto_library and such)?

Reply:
The reason the others are done here is because SysRootPath may throw, so we do not want to throw in the constructor and just during actual generation.

Reply:
I didn't realize it was done there as well. Thank you. Updated.